server/gameconnection.py

```diff
@@ -471,11 +479,15 @@ async def handle_game_state(self, state: str):
             if self.player.state != PlayerState.HOSTING:
                 return
 
-            self._logger.info(
-                "Launching game %s in state %s",
-                self.game,
-                self.game.state
-            )
+            if self.game.state is not GameState.LOBBY:
+                self._logger.warning(
+                    "Trying to launch game %s in invalid state %s",
+                    self.game,
+                    self.game.state
+                )
+                return
+
+            self._logger.info("Launching game %s", self.game)
```

Comment:
I think including which player is launching the game could be useful.

Reply:
It's always the host, and the host is included in the string representation of `self.game`